guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,293 @@
+
+```json
+[
+  {
+      "id": "495925",
+      "title": "Doraemon the Movie:Nobita's Treasure Island",
+      "director": "Fujiko Fujio",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/cmJ71gdZxCqkMUvGwWgSg3MK7pC.jpg",
+      "overview": "The story is based on Robert Louis Stevenson's Treasure Island novel.",
+      "release_date": 1520035200
+  },
+  {
+      "id": "329996",
+      "title": "Dumbo",
+      "director": "Tim Burton",
+      "genre": "Science fiction",
+      "poster": "https://image.tmdb.org/t/p/w1280/279PwJAcelI4VuBtdzrZASqDPQr.jpg",
+      "overview": "A young elephant, whose oversized ears enable him to fly, helps...",
+      "release_date": 1553644800
+  },
+  {
+      "id": "458723",
+      "title": "Us",
+      "director": "Jordan Peele",
+      "genre": "Horror",
+      "poster": "https://image.tmdb.org/t/p/w1280/ux2dU1jQ2ACIMShzB3yP93Udpzc.jpg",
+      "overview": "Husband and wife Gabe and Adelaide Wilson take their...",
+      "release_date": 1552521600
+  },
+  ...
+]
+```
+
+To be able to facet search on `director` and `genre`, you would declare faceted attributes as follows:
+
+```bash
+$ curl \
+  -X POST 'http://localhost:7700/indexes/movies/settings' \
+  --data '{
+      "attributesForFaceting": [
+          "director",
+          "genres"
+      ]
+  }'
+```
+
+## Using facets
+
+By introducing facets to MeiliSearch, new search query parameters were also added:
+
+- `facetFilters`:  Narrows the selection on which to search.
+- `facetsDistribution`: Returns the number of matching documents distributed amongst all the values of a given facet.
+
+### The facet filters
+
+Because facets relate specifically to a set of documents, they give an overview of the records and help to understand what kind of information can be searched. Facets then help users navigate through their search results.
+
+![Amazon movies](/faceted-search/facets-akira.png)
+> You can select multiple categories on Amazon.
+
+They can filter on facets to narrow down their results based on criteria with the `facetFilters` attribute.
+
+[Learn more about `facetFilters` in the search parameters](/guides/advanced_guides/search_parameters.md#facet-filters)
+
+### The facets distribution
+
+Since the users can have a visual clue about the range of categories available in the UI, they can easily know how many search results are found for each category.
+
+In the example below, on [IMDb](https://www.imdb.com), the number in parentheses matches how many search results each facet relates to.
+
+![IMDb facets](/faceted-search/facets-imdb.png)
+> If a user searches Fantasy, Animation, Adventure movies, and TV shows, they will know there are 826 TV series and 137 video games matching their criteria.
+
+To get the facets distribution, you have to specify a list of facets for which to retrieve the count of matching documents using the `facetsDistribution` attribute.
```

Comment:
```suggestion
Since the users can have a visual clue about the range of categories available in the UI, they can easily know how many search results are found for each category.

To get the facets distribution, you have to specify a list of facets for which to retrieve the count of matching documents using the `facetsDistribution` attribute.
```